### PR TITLE
Minor FSTree improvements

### PR DESCRIPTION
**Pull request by fyrchik**

### CODE REVIEW

File: pkg/local_object_storage/blobstor/fstree/fstree.go [hunk 1]
```diff
@@ -71,27 +71,27 @@ func stringifyAddress(addr oid.Address) string {
 	return addr.Object().EncodeToString() + "." + addr.Container().EncodeToString()
 }
 
-func addressFromString(s string) (*oid.Address, error) {
-	ss := strings.SplitN(s, ".", 2)
-	if len(ss) != 2 {
-		return nil, errors.New("invalid address")
+func addressFromString(s string) (oid.Address, error) {
+	i := strings.IndexByte(s, '.')
```
Comment (carpawell) [inline pkg/local_object_storage/blobstor/fstree/fstree.go:75]: some bench to commit/PR decs about enhancement?

Comment (fyrchik) [reply pkg/local_object_storage/blobstor/fstree/fstree.go:75]: Fixed

**Review by acid-ant — APPROVED** [review]

**Comment by acid-ant** [comment]
Changelog needs to be updated.

**Comment by fyrchik** [comment]
IMO it isn't worth a line in changelog -- purely internal opitimizations.

**Pull request merged**